### PR TITLE
updated to UIKit 2.18.0

### PR DESCRIPTION
**Pull request by tryIO**
Let's see if we need another rebase or not.

### CODE REVIEW

**Comment by jamesward** [comment]
Yeah, looks like you need to rebase on master.

**Comment by tryIO** [comment]
it should be ok now!

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>uikit</artifactId>
-    <version>2.17.0</version>
+    <version>2.18.0</version>
```
Comment (jamesward) [inline pom.xml]: This needs to be `2.18.0-SNAPSHOT` since Maven does the actual release.

**Comment by tryIO** [comment]
oh gosh! sorry I fixed!

**Pull request merged**